contracts/facets/EmbalmerFacet.sol

```diff
@@ -126,15 +133,15 @@ contract EmbalmerFacet {
 
             // Validate archaeologist profile value requirements are met
             LibUtils.revertIfDiggingFeeTooLow(arch.diggingFee, arch.archAddress);
-            LibUtils.revertIfResurrectionTimeTooFarInFuture(sarcophagus.resurrectionTime, arch.archAddress);
```

Comment:
realizing that we could have left this in and omitted the signature of the agreedMaximumRewrapInterval, but hunch is that this new way probably saves gas

Reply:
could also make argument we could now remove `maxRewrapInterval` from the archaeologist profile and have the arch service provide this data to the embalmer web app, since the signature now acts as the on-chain validation of this value.

Reply:
https://github.com/sarcophagus-org/sarcophagus-v2-contracts/issues/73